src/pages/AgentNew.tsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate, Link } from 'react-router-dom'
+import { Button, FloatingLabel, Form } from 'react-bootstrap'
+import icon from '../images/icon.svg'
+
+export default function AgentNew() {
+  const navigate = useNavigate()
+
+  return (
+    <div>
+      <a role="button" onClick={() => navigate(-1)}>
+        <i className="bi bi-chevron-left align-middle" />
+        Back
+      </a>
+      {/* place text horizontal to 'Back' nav link? */}
+      <h2 className="mb-4 fw-normal text-center">Add listing</h2>
+      {/* switch to officer icon */}
+      <div className="mb-4 text-center">
+        <img src={icon} alt="ProPro logo" style={{ width: 50 }} />
+      </div>
+      <div className="p-4 text-start">
+        <h2 className="mb-2">Add an agent</h2>
+        <h5 className="mb-5 fw-normal">
```

Comment:
For standard blocks of text, we should use a p tag here
```suggestion
        <p className="mb-5">
```